### ISSUE TITLE
Sorting columns by time descending

### ISSUE THREAD

**Issue by slyoldfox** [MaxBelkov/visualsyslog#12]
Would it be possible to add a sorting based on the columns. Basically only "time" would be interesting to sort descending. New lines to be added on top - instead of bottom.

**Comment by ThomasCr** [comment]
Yes, I also wish to see this too. I dont know why and how, but my time colum is not sortet. So I have 

```
Feb 20 08:06:05
Feb 20 08:13:08
Feb 20 00:00:24
```

**Comment by GuyManDude** [comment]
Another request for being able to sort the time column by descending please.
